resources/public/api/conf/1.0/schemas/retrieveBusinessDetails/retrieve_business_details_response.json

```diff
@@ -11,7 +11,7 @@
     },
     "typeOfBusiness": {
       "type": "string",
-      "description": "The type of business income source, one of: `self-employment`, `uk-property` or `foreign-property`.",
+      "description": "The type of business income source, one of: `self-employment`, `uk-property`, `foreign-property` or `property-unspecified`.",
       "example": "self-employment"
```

Comment:
Can this `"example": "self-employment"` be replaced with an enum thing like we have with the HATEOAS `rel` and `method` fields? It'd look something like this:
```
"typeOfBusiness": {
  "type": "string",
  "description": "The type of business income source.",
  "enum": [
    "self-employment",
    "uk-property",
    "foreign-property",
    "property-unspecified"
  ]
}
```

Reply:
Same for the accountingType field - something like:
```
"accountingType": {
  "description": "CASH if you used the cash basis, money actually received and paid out, to calculate your income and expenses. Otherwise ACCRUALS.",
  "type": "string",
  "enum": [
    "CASH",
    "ACCRUALS"
  ]
}
```